rtc/introduction.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: RTC Introduction
+title_nav: Introduction
+description: Introduction of what RTC is and its capabilities
+keywords: rtc introduction overview
+---
+
+{% assign beta_feature = "The RTC plugin" %}
+{% assign pre-release_type = "Closed Beta" %}
+{% include misc/beta-note.md %}
+{{site.requires_5_6v}}<br>
+{{site.premiumplugin}}<br>
+
+## Introduction
+
+{% include misc/rtc-description.md %}
+
+## JSON Web Token based authentication
+
+Some cloud services for {{site.productname}} require setting up JSON Web Token (JWT) authentication. JWTs are a common solution for communicating user authorization for web services. JWTs are used to communicate to {{site.productname}} that the user has been authorized to access {{site.cloudname}} services.
+
+For general information on JSON Web Tokens, visit: [https://jwt.io/](https://jwt.io/).
+
+For information on using JWT authentication with the RTC plugin, see: [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/).
+
+## Presence API
+
+The RTC plugin comes with a presence API that enables you to track when users enter/leave the session. The only user information shared over the wire is the user id stored in the JWT `sub` claim. Other details like the users full name is resolved locally so the {{site.cloudname}} will never see who is actually connecting. There are currently 8 distinct caret colors assigned to each connecting user. More than 8 users can still connect but then the colors will be reused.
```

Comment:
```suggestion
The RTC plugin comes with a presence API that enables you to track when users enter/leave the session. The only user information shared over the wire is the user id stored in the JWT `sub` claim. Other details like the user's full name is resolved locally so the {{site.cloudname}} will never see who is actually connecting. There are currently 8 distinct caret colors assigned to each connecting user. More than 8 users can still connect but then the colors will be reused.
```

Also, not sure how to fix this but it makes it sound like each user gets 8 colours, rather than the first 8 users get unique colours and then it cycles

Reply:
resolved in https://github.com/tinymce/tinymce-docs/pull/1778/commits/9cc200b9775691b5106c568717f3069b51dbb46e